hack/e2e.sh

```diff
@@ -131,6 +132,9 @@ while getopts "h?" opt; do
     esac
 done
 
+# TODO(mauriciopoppe): remove this line after retry
```

Comment:
what is retry?

Reply:
I’m going to remove this line, and lease ignore it